### PR TITLE
fix link

### LINKED ISSUE #25: rewrite zhihu link in answer/article
for example, <https://www.fxzhihu.com/question/610789734/answer/3111205257> , at the bottom, another zhihu link there.

### PR DESCRIPTION
**Pull request by asukaminato0721**
fix #25

### CODE REVIEW

**Comment by Colerar** [comment]
It is best to make the redirected new domain configurable instead of constant `fxzhihu.com`. It's useful for testing. The links would jump to localhost dev server instead of production server after configuring.

File: src/lib.ts [hunk 1]
```diff
@@ -128,23 +128,17 @@ export async function TransformUrl(url: string, isLocal: boolean) {
     try {
       const questionMatch = patterns.question.exec(urlString);
       if (questionMatch) {
-        return isLocal
-          ? `http://localhost:8787/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`
-          : `https://www.fxzhihu.com/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`;
+        return (isLocal ? `http://localhost:8787` : `https://www.fxzhihu.com`) + `/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`
```
Comment (frostming) [inline src/lib.ts]: Is it possible to get the URL from `Env`

Comment (asukaminato0721) [reply src/lib.ts]: fixed

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ https://greasyfork.org/zh-CN/scripts/510234-zhihu-link-fixer
 ## 开发
 
 ```
-pnpm dev
```
Comment (frostming) [inline README.md]: Should add the option to `package.json` instead of exposing it outside.

**Review by frostming — APPROVED** [review]

**Pull request merged**